generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -203,8 +203,10 @@ export const <%= entityReactName %>Update = (props: I<%= entityReactName %>Updat
         , <%= rel.relationshipFieldNamePlural %>: mapIdList(values.<%= rel.relationshipFieldNamePlural %>)
         <%_ }) _%>
       }
-      <%_ if (uniqueRelationFields.has('users')) { _%>
-      entity.user = values.user;
+      <%_ if (uniqueRelationFields.has('users') && authenticationType === 'oauth2' && applicationType === 'gateway' && dto !== 'mapstruct') { _%>
+        <%_ relationships.filter(rel => rel.relationshipType === 'one-to-one' && rel.ownerSide === true && rel.otherEntityName === 'user').forEach(rel => { _%>
```

Comment:
`_%>\n<%_ ...`
There's no need for two tags in this case

Reply:
Forget it... I must be tired because I actually believed that something like that:
```
<%_ ... { _% >
  <%_ ... { _% >
    ...
  <%_ ... } _% >
<%_ ... } _% >
```

was equal to
```
<%_ ... {
   ... { _% >
    ...
  <%_ ... }
... } _% >
```

apparently this isn't the case :/